po/fr.po

```diff
@@ -686,7 +684,7 @@ msgstr "Paramétrer maintenant"
 
 #: data/interface/welcome.ui:76
 msgid "Welcome to Cronopete"
-msgstr "Bienvenu à Cronopete"
+msgstr "Bienvenu dans Cronopete"
```

Comment:
`Bienvenue sur Cronopete` may be better :thinking:

Reply:
I don't think. In french, people use generally "dans" for an action linked to a software. "Dans ton logiciel de mail, tu clique ici"

Reply:
I'm a native speaker ... But anyway it's only a matter of opinion ...

Reply:
Je ne sais pas. Spontanément je dirais effectivement "dans" pour un logiciel. Mais effectivement "sur" pourrait avoir du sens comme dans "bienvenue sur Mars".  Mais cela n'a sans doute pas beaucoup de détails. Et c'est toujours mieux que "à" qui est un contresens.

Reply:
Exactly :+1: